### DISCUSSION TITLE
Web editor: vim emulation stopped working

### DISCUSSION THREAD

**Discussion by harveyr** [community/community#6187] [Codespaces] [answered]
Hey all! 

When I first started using it in the github.dev editor, the Vim extension (vscodevim) worked just fine as soon as I installed it. I'd open a file, and as soon as the extensions loaded, I'd be in normal mode.

It stopped working a few days ago. It _seems_ to have happened at the same time that I was offered to sync my settings, which I accepted. (I could be wrong about that timing.) I've since disabled sync, wiped out my settings in the github.dev editor, and uninstalled/reinstalled the Vim extension several times. I can't get it to work again.

Thoughts? Thanks!

# Updates

I do see an error in the Worker Extension Host logs:
```
[2021-10-13 16:28:59.225] [exthost.log] [error] Activating extension vscodevim.vim failed due to an error:
[2021-10-13 16:28:59.267] [exthost.log] [error] {}
```
Possibly related issues in the vscodevim project:

- https://github.com/VSCodeVim/Vim/issues/7150
- https://github.com/VSCodeVim/Vim/issues/6826

**Comment by bamurtaugh** [comment, marked as answer]
Thanks for reporting @harveyr. Based on https://github.com/VSCodeVim/Vim/issues/6826#issuecomment-946331874, it looks like this should be fixed in 1.29.1 of the extension. Does it work for you?

**Reply by harveyr** [reply]
Sure does, thank you!
